secant/UIComponents/Backgrounds/ScreenBackground.swift

```diff
@@ -10,10 +10,7 @@ import SwiftUI
     A Vertical LinearGradient that takes an array of Colors and renders them vertically in a centered fashion mostly used as a background for Screen views..
 */
 struct ScreenBackground: View {
-    var colors = [
-        Asset.Colors.ScreenBackground.gradientStart.color,
-        Asset.Colors.ScreenBackground.gradientEnd.color
-    ]
+    var colors: [Color]
```

Comment:
GitHub doesn't make this easy to see but can we add as pace below this variable?